recipes/fluidfoam/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "fluidfoam" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5d85adfda2e9cedfb71f1206d98a22ae644dda7ec7e71c22760b670c2d7890f8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - numpy >=1.11
+    - scipy >=0.17
+    - matplotlib >=1.5
+
+test:
+  imports:
+    - fluidfoam
+
+about:
+  home: https://bitbucket.org/fluiddyn/fluidfoam
+  license: GNU General Public v2 (GPLv2)
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: GPL-2.0
```